project/Mima.scala

```diff
@@ -0,0 +1,10 @@
+import com.typesafe.tools.mima.core._
+import com.typesafe.tools.mima.core.ProblemFilters._
+
+object Mima {
+  val ignoredABIProblems: Seq[ProblemFilter] = {
+    Seq(
+      exclude[MissingClassProblem]("scalafix.package$XtensionRewriteCtx")
```

Comment:
It's a way to document binary breaking changes. `sbt mimaReportBinaryIssues` compares the classfiles of the current HEAD with the classfiles of the latest published release to Maven and reports issues such as "missing class/method". It's possible to ignore those issues by acknowledging them like this, to indicate you the breaking change is OK.